tests/robot/4_check_edi.robot

```diff
@@ -58,11 +62,11 @@ Check EDI deploy with scale to 1
 Check EDI deploy with scale to 2
```

Comment:
Can we check actual scale here?

Reply:
@aliaksandr-d what do you mean? We deploy with scale = 2 flag, and check by "inspect" command  that it scaled to 2 after deploy.